tools/src/dbgen/config.dhall

```diff
@@ -1,3 +1,4 @@
 { wallet_spec = { account_spec = { addresses = 100 }, accounts = 1 }
+, fakeUtxo    = { fromAddress = 100, amount = 10000 }
```

Comment:
Interestingly you took a similar path of the one I am proposing here, that's good, it means we are not too far off to a good solution 😉